spec/components/table_spec.rb

```diff
@@ -153,4 +153,25 @@ def component_name
     assert_select ".js-gem-c-table__filter"
     assert_select ".js-gem-c-table__message"
   end
+
+  it "accepts a width value for head cells" do
+    render_component(
+      head: [
+        {
+          text: "Month you apply",
+          width: "one-third",
+        },
+      ],
+      rows: [
+        [
+          {
+            text: "January",
+          },
+        ],
+      ],
+    )
+
+    # This needs [class=...] attribute syntax as Nokogiri doesn't like the `!` in the class name
+    assert_select ".govuk-table th[class='govuk-table__header govuk-!-width-one-third']", "Month you apply"
```

Comment:
I think you can also do the following to match classes with a `!` in them, for information. No need to change this line though.

```
assert_select '.govuk-\!-margin-bottom-6'
```

Reply:
Oh cool – I think I _did_ try that but it but it still complained, maybe because it got tripped up by the sibling class 👍